app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -58,17 +59,22 @@ public Media createFromParcel(Parcel parcel) {
     protected String licenseUrl;
     protected String creator;
     protected ArrayList<String> categories; // as loaded at runtime?
+    protected ArrayList<String> depictions;
     protected boolean requestedDeletion;
     private Map<String, String> descriptions; // multilingual descriptions as loaded
+    private HashMap<String, String> captions;
```

Comment:
Explain what is the key and what is the value. Add an example with 2 or 3 entries.

Reply:
Done